From 0bf5405e979f9260913391c5338e3e6afeb47312 Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Sat, 18 Feb 2017 16:42:58 +0100
Subject: [PATCH 152/206] Focus only when tapping the preview

Calling onSingleTapUp() from onScroll() is causing Snap to issue
a focus request for every touch movement, including the gestures
to open the menu and the gallery.

Change-Id: If913607735add13c1af8e0882748d4800e4e10f9
---
 src/com/android/camera/PreviewGestures.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/com/android/camera/PreviewGestures.java b/src/com/android/camera/PreviewGestures.java
index 8a0c09b80..48d972f7c 100644
--- a/src/com/android/camera/PreviewGestures.java
+++ b/src/com/android/camera/PreviewGestures.java
@@ -101,9 +101,9 @@ public class PreviewGestures
                 if (mCaptureUI != null)
                     mCaptureUI.openSettingsMenu();
                 return true;
-            } else {
-                return onSingleTapUp(e2);
             }
+
+            return false;
         }
 
         private boolean isLeftSwipe(int orientation, int deltaX, int deltaY) {
-- 
2.17.1

